Sources/Kitura/RouterResponse.swift

```diff
@@ -269,6 +269,54 @@ public class RouterResponse {
 
         return self
     }
+    
+    #if os(Linux)
+        typealias JSONSerializationType = LclJSONSerialization
+    #else
+        typealias JSONSerializationType = JSONSerialization
+    #endif
+    
+    /// Send JSON.
+    ///
+    /// - Parameter json: The array to send in JSON format.
+    /// - Returns: this RouterResponse.
+    @discardableResult
+    public func send(json: [Any]) -> RouterResponse {
+        guard !state.invokedEnd else {
+            Log.warning("RouterResponse send(json:) invoked after end() for \(self.request.urlURL)")
+            return self
+        }
+        do {
+            let jsonData = try JSONSerializationType.data(withJSONObject: json, options:.prettyPrinted)
```

Comment:
Would consumers of this would normally be applications, and if so, should we pretty print? I think most browsers will pretty-fy json automatically.